src/api-reference/common/list-item/v4.list-item-bulk.markdown

```diff
@@ -0,0 +1,494 @@
+
+#### Response
+
+```shell
+HTTP/1.1 200
+concur-correlationid: 46d1b015-635e-4bca-be76-2ed39c122065 
+content-length: 289
+content-type: application/json;charset=UTF-8
+location: https://us.api.concursolutions.com/list/v4/lists/530ae443-3b22-a44c-8211-c66d96c58d9e/bulk
+date: Fri,26 Aug 2022 17:11:35 GMT 
+cache-control: no-cache, private
+```
+
+Example 1:
+```json
+{
+  "status": "SUCCESS",
+  "recordsSucceeded": 1,
+  "recordsFailed": 0,
+  "errors": []
+}
+
+```
+
+Example 2:
+```json
+{
+  "status": "SUCCESS",
+  "recordsSucceeded": 1,
+  "recordsFailed": 0,
+  "errors": []
+}
+
+```
+
+Example 3:
+```json
+{
+  "status": "PARTIAL_SUCCESS",
+  "recordsSucceeded": 2,
+  "recordsFailed": 1,
+  "errors": [
+    {
+      "message": "The list item has been deleted",
+      "listItem": {
+        "code": "ITEM",
+        "value": "ITEM"
+      }
+    }
+  ]
+}
+```
+
+## <a name="schema"></a>Schema
+
+### <a name="schema-bulk-list-item-create-request"></a>Bulk List Item Create Request
+|Name|Type|Format|Description|
+|---|---|---|---|
+|`requests`|`list`|[`BulkListItemCreateRequestPart`](#schema-bulk-list-item-create-request-part)|The requests in the bulk list create request|
+
+
+### <a name="schema-bulk-list-item-create-request-part"></a>Bulk List Item Create Request Part
+|Name|Type|Format|Description|
+|---|---|---|---|
+|`shortCode`|`string`|-|List item short code|
+|`value`|`string`|-|List item value|
+|`parentCode`|`string`|-|The long code of parent list item|
+
+
+### <a name="schema-bulk-list-item-update-request"></a>Bulk List Item Update Request
+|Name|Type|Format|Description|
+|---|---|---|---|
+|`requests`|`list`|[`BulkListItemUpdateRequestPart`](#schema-bulk-list-item-update-request-part)|The requests in the bulk list update request|
+
+### <a name="schema-bulk-list-item-update-request-part"></a>Bulk List Item Update Request Part
+|Name|Type|Format|Description|
+|---|---|---|---|
+|`code`|`string`|-|List item long code|
+|`value`|`string`|-|List item value|
+|`deleted`|`boolean`|-|List item deleted status|
+
+### <a name="schema-bulk-list-item-response"></a>Bulk List Item Response
+|Name|Type|Format|Description|
+|---|---|---|---|
+|`status`|`string`|-|Response status ( Enum:`SUCCESS`, `PARTIAL_SUCCESS`, `FAILURE`)|
+|`recordsSucceeded`|`int`|-|Number of list items successfully created for the list|
+|`recordsFailed`|`int`|-|Number of list items failed to be created for the list|
+|`errors`|`list`|[`BulkListError`](#schema-bulk-list-error)|List of error codes and list items failed to be created|
+
+
+### <a name="schema-bulk-list-error"></a>Bulk List Error
+|Name|Type|Format|Description|
+|---|---|---|---|
+|`message`|`string`|-|The detailed error message|
+|`listItem`|[`BulkListItemRequest`](#schema-bulk-list-item-request)|-|List item corresponding to the error|
+
+### <a name="schema-bulk-list-item-request"></a>Bulk List Item Request
+|Name|Type|Format|Description|
+|---|---|---|---|
+|-|-|-|	List item corresponding to the error|
```

Comment:
This seems confusing. I'm wondering if we should have the Type of the "Bulk List Error" be 
```
[`BulkListItemCreateRequest`](#schema-bulk-list-item-create-request) or [`BulkListItemUpdateRequest`](#schema-bulk-list-item-update-request)
```
?